### PR TITLE
#29 Configuration per RealmModule

### PR DESCRIPTION
**Pull request by magillus**
Added support for RealmModule defined classes per configuration - without kotlin reflection (using Java)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -3,6 +3,7 @@
 buildscript {
     ext.kotlin_version = '1.1.51'
     repositories {
+        google()
```
Comment (magillus) [inline build.gradle]: master do not build without this

**Comment by vicpinm** [comment]
Hi @magillus, your PR looks good, next week I will review it more carefully and I will merge into master, thanks you very much!

**Comment by magillus** [comment]
If need some update and more comments or java doc let me know

**Comment by magillus** [comment]
I will update conflict and push soon

**Comment by vicpinm** [comment]
@magillus don't you worry about that, I currently reviewing your PR and I will merge it into master soon.

**Comment by magillus** [comment]
ok, thanks - just merged and pushed.

**Comment by vicpinm** [comment]
@magillus the only thing I'm not sure why is changed is about this lines in Application class:

// delete all data		
 Realm.deleteRealm(Realm.getDefaultConfiguration())		
 Realm.deleteRealm(userAddressConfig)

why did you remove them?

**Comment by magillus** [comment]
That is for fresh start on each application test run.
I run into issues with my testing so that `userAddressConfig` classes were stored in default config and stayed there when the issue was fixed - this make sure that every run of the app has clean DBs.

**Comment by vicpinm** [comment]
Okey, I have merged it into master. I will upload a new version to maven soon!

Thanks you very much.

**Pull request closed without merging**